proto/buf/alpha/registry/v1alpha1/recommendation.proto

```diff
@@ -32,6 +34,19 @@ message RecommendedTemplate {
   string owner = 1;
   string name = 2;
   string description = 3;
+  string template_id = 4;
+}
+
+// RecommendingRepository is the information needed to configure a repository recommendation
+message RecommendingRepository {
```

Comment:
this name could be better I guess, but I cannot come up with one

Reply:
Haha I'm trying to think of one but I think you're right I don't know 😂.

Reply:
It would technically be possible to use the message `RecommendedRepository` here. I'm not sure that the two added messages are much cleaner than documenting that `SetRecommendedRepositories` only requires `repository_id` and `description` to be set. Have we discussed such a case before?

Reply:
> I think we generally use the resources only in return messages, and take whatever the required input to creation/deletion is explicitly.

That seems worth sticking to.

`SetRecommendedRepository` seems like a better fit to me than `RecommendingRepository` - what is your take, @cyinma?

Reply:
it seems good, will update it